points/module.py

```diff
@@ -164,103 +141,10 @@ async def on_message(self, message):
-            return
-
-        if getattr(reaction.message, "guild", None) is None:
-            return
-
-        # add points
-        guild_id = reaction.message.guild.id
-
-        value = random.randint(LIMITS_REACTION[0], LIMITS_REACTION[1])
-
-        Points._handle_points(
-            guild_id, user.id, self.stats_reaction, TIMER_REACTION, value
-        )
-
-        if str(reaction) not in ("⏪", "◀", "▶"):
-            return
-
-        tc = TranslationContext(guild_id, reaction.message.author.id)
-
-        if (
-            len(reaction.message.embeds) != 1
-            or type(reaction.message.embeds[0].title) != str
-            or (
-                not reaction.message.embeds[0].title.startswith(_(tc, "Points 🏆"))
-                and not reaction.message.embeds[0].title.startswith(_(tc, "Points 💩"))
-            )
-        ):
-            return
-
-        embed = reaction.message.embeds[0]
-
-        # get ordering
-        if embed.title == _(tc, "Points 💩"):
-            order = BoardOrder.ASC
-        else:
-            order = BoardOrder.DESC
-
-        # get current offset
-        if ", " in embed.fields[0].name:
-            offset = int(embed.fields[0].name.split(" ")[-1]) - 1
-        else:
-            offset = 0
-
-        # get new offset
-        if str(reaction) == "⏪":
-            offset = 0
-        elif str(reaction) == "◀":
-            offset -= 10
-        elif str(reaction) == "▶":
-            offset += 10
-
-        if offset < 0:
-            return await utils.Discord.remove_reaction(reaction.message, reaction, user)
-
-        users = UserStats.get_best(guild_id, order, 10, offset)
-        value = Points._get_board(reaction.message.guild, user, users)
-        if not value:
-            # offset too big
-            return await utils.Discord.remove_reaction(reaction.message, reaction, user)
-
-        if order == BoardOrder.DESC:
-            table_name = _(tc, "Best {limit}")
-        else:
-            table_name = _(tc, "Worst {limit}")
-
-        name = table_name.format(limit=10)
-
-        if offset:
-            name += _(tc, ", position {offset}").format(offset=offset + 1)
-
-        embed.clear_fields()
-        embed.add_field(name=name, value=value, inline=False)
-
-        # if the user is not present, add them to second field
-        if user.id not in [u.user_id for u in users]:
-            author = UserStats.get_stats(guild_id, user.id)
-
-            embed.add_field(
-                name=_(tc, "Your score"),
-                value="`{points:>8}` … {name}".format(
-                    points=author.points,
-                    name="**" + utils.Text.sanitise(user.display_name) + "**",
-                ),
-                inline=False,
-            )
-
-        await reaction.message.edit(embed=embed)
-        await utils.Discord.remove_reaction(reaction.message, reaction, user)
-
     # Helper functions
 
     @staticmethod
-    def _get_board(
+    def _get_page(
```

Comment:
This function has `ctx = TranslationContext`, can you update that to `tc = TranslationContext`, please? GitHub does not allow me to comment on the affected line, it's just at the top.